### PR TITLE
Move res

### PR DESCRIPTION
**Pull request by eivindjahren**
**Issue**
More work towards resolving #3595 

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: docs/experimental/plugin_system.rst [hunk 1]
```diff
@@ -135,7 +135,7 @@ Full example:
 
 .. code-block:: python
 
-   from res.enkf import ErtScript
+   from ert import ErtScript
    from ert.shared.plugins.plugin_manager import hook_implementation
```
Comment (oyvindeide) [inline docs/experimental/plugin_system.rst:139]: Will this be another PR to align?

Comment (eivindjahren) [reply docs/experimental/plugin_system.rst:139]: This is already in the `ert/__init__.py`, but yes, I will create another PR to align this with semeio.
